projects/plugins/jetpack/extensions/blocks/blogroll/edit.js

```diff
@@ -46,14 +46,19 @@ export function BlogRollEdit( { className, attributes, setAttributes, clientId }
 		} ),
 	} );
 
+	const renderBlogrollAppender = () => {
+		if ( subscriptions && subscriptions.length ) {
+			return () => <BlogrollAppender subscriptions={ subscriptions } clientId={ clientId } />;
+		}
+		return null;
+	};
```

Comment:
What if you have none? You won't be able to append?

Reply:
Im going to revert this and handle in separate PR